### PR TITLE
Fixes bombs on Reebe being an instant win

### PR DESCRIPTION
**Pull request by Robustin**
:cl: Robustin
fix: Fixed a bug where detonating maxcaps, especially multiple maxcaps, on Reebe guaranteed that everyone would die and thus the Clock Cult would immediately lose; Reebe maxcap is now 2/5/10. 
/:cl:

I'm a good boy. 

Yes this is a self-inflicted nerf, maxcap is now effectively 2/5/10 on Reebe. Currently, a single maxcap on the cult side of Reebe basically guarantees they'll all be crit. Multiple maxcaps guarantees death.

As much as I would like "emergent gameplay" around TTV's, its not realistic to ask an off-station cult with no science access to shut down Toxins every game, nor is it particularly fair that the cult could hypothetically try to rush them at roundstart at which point its all but impossible to stop them from building their own maxcap bonanza. 

I love bombs so you know it's gotta be a serious problem for me to do this.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
Calling it a fix is quite a stretch

**Comment by Robustin** [comment]
It's a joke because penalizing coders for gameplay improvements that require ~15 seconds of review is asinine.

**Comment by GuyonBroadway** [comment]
Now we just need to stop BoH bombing rebee working properly.

**Comment by Supermichael777** [comment]
Teleport in and rob toxins the trip after you break the servers and grab the console board

**Comment by Armhulen** [comment]
Michael is correct

**Comment by Robustin** [comment]
If someone is working toxins, good luck, you risk out'ing the cult extremely early.

Even if you succeed it just flips the equation. Steal the TTV's and there's almost no way to stop the CC from being able to get the plasma/oxygen/pumps to make maxcaps. Maxcaps placed at the southern edge is GG crew.

I really don't like maxcaps on Reebe since there's so little risk involved once you've got the bombs. Any other team mode requires a lot of strategic considerations and risk in executing a bombing. With CC both sides start the fight in a single area and it's virtually impossible to prevent the other side from using them for an auto-win if they bring them to Reebe.

File: code/datums/explosion.dm [hunk 1]
```diff
@@ -60,6 +61,13 @@ GLOBAL_LIST_EMPTY(explosions)
 		light_impact_range = min(GLOB.MAX_EX_LIGHT_RANGE, light_impact_range)
 		flash_range = min(GLOB.MAX_EX_FLASH_RANGE, flash_range)
 		flame_range = min(GLOB.MAX_EX_FLAME_RANGE, flame_range)
+		
+	if(!ignorecap && epicenter.z == ZLEVEL_CITYOFCOGS)
```
Comment (Cyberboss) [inline code/datums/explosion.dm]: I mean, it's a shim, but I can't think of a better way at the moment so whatever

**Review by imtakingabreakdontatme — APPROVED** [review]

**Comment by KorPhaeron** [comment]
@Xhuis

**Comment by Xhuis** [comment]
I don't agree with this change. That said, I'm willing to see how it plays out, but I'd like some testmerges first to ensure that bombs are still effective weapons in practice and not firecrackers nor miniature nukes. I want them to be a completely viable strategy against servants, if not the dominant one.

This change is also somewhat hard to document, and I can foresee a lot of angry people wondering why their formulas aren't working. Maybe you could display a large chat message to anyone entering Reebe with explosives? Here's some pseudocode:
```
/obj/blah/reebe_rift
     var/static/explosives_typecache = list() //I don't know how typecaches work

/obj/blah/reebe_rift/blah_the_guy(atom/guy)
     var/list/guy_stuff = guy.GetAllContents()
     for(var/obj/item/I in guy_stuff)
          if(is_type_in_typecache(I, explosives_typecache))
               to_chat(guy, "<span class='userdanger'>You feel your [I.name] pulse with freezing cold; its effectiveness is reduced here!</span>")
               break //We already told him once, no need to spam him for all fifteen of his maxcaps

**Comment by Jazaen** [comment]
They should still be viable. 2/5/10 is still more than enough to tactically nuke good part of cult defenses if they all pile on top of a breach, but it prevents one maxcap from, well, doing what Robustin described.

**Comment by Robustin** [comment]
Again, in testing I put humans on the outer edge of Reebe past the "line", then I dropped a 5/10/20, JUST ONE, in the middle - and every cultist was in crit except for the ones in the far corners and the ones behind the ark. 

2-3x was overkill but made sure nobody survived if they were anywhere on the cult side. 

At half the power a single bomb will still breach and pose a lethal threat to anyone in screen range, stack bombs and you'll still be able to wipe out a good quarter of Reebe.

**Comment by Supermichael777** [comment]
make reebe naturally pressurized

**Comment by ChangelingRain** [comment]
it already is

**Comment by Robustin** [comment]
Yea its not a pressure or hull breach issue its more along the lines of "3 light ex_acts are still super lethal".

**Comment by kevinz000** [comment]
lmao i ded but :+1:

**Comment by Supermichael777** [comment]
because the very edge of an explosion should totaly have random delimbs right guys its balanced invisible shrapnel

**Review by Cyberboss — DISMISSED** [review]
I agree with @Xhuis this needs some sort of notification that bombs are nerfed on Reebe if someone brings them over

**Comment by Robustin** [comment]
I wasn't worried because the bomb still explodes and kills or crits anyone on screen, it might not be everything you hoped but people will figure it out and adapt just like they did with blobs being bombproof. 

I'll add a message explicitly for people with TTV's for now, since I'm not aware of any other crew maxcap device except for canister bombs but anyone who understands those dark arts has already memorized explosive code anyway.

**Comment by ExcessiveUseOfCobblestone** [comment]
> Even if you succeed it just flips the equation. Steal the TTV's and there's almost no way to stop the CC from being able to get the plasma/oxygen/pumps to make maxcaps. Maxcaps placed at the southern edge is GG crew.

Is robustin about to make a breakthrough? What could be the common thread here....

**Comment by Xhuis** [comment]
I had requested this be testmerged first.

**Pull request merged**